megamek/src/megamek/common/EntityListFile.java

```diff
@@ -105,7 +105,7 @@ private static String formatSlot(String index, Mounted mount,
                 if (mount.getEntity().usesWeaponBays() 
                         || mount.getEntity() instanceof Dropship) {
                     output.append("\" capacity=\"")
-                        .append(String.valueOf(mount.getAmmoCapacity()));
+                        .append(String.valueOf(mount.getSize()));
```

Comment:
This is related?

Reply:
No, but it's listed as deprecated so I figured I'd change it while I was there.